### DISCUSSION TITLE
Battery Level refresh interval

### DISCUSSION THREAD

**Discussion by maximumstruggle** [flobz/psa_car_controller#590] [Q&A]
Hello,

I apologize if it is witten somewhere but I couldn't find it : can I change the battery level refresh interval to a longer value ? I don't have a fast charger at home so I don't need it to be refreshed every minute.

**Comment by papageigit** [comment]
The interval depends on if you are AC or DC charging. In my experience when AC charging, it updates every 15 minutes, while when DC charging, it is updating in realtime.

**Reply by maximumstruggle** [reply]
Thank you for your answer. It's weird then, mine is updating every 2 min on AC charging. I don't know why :

```
2023-07-26 13:56:50,805 :: INFO :: charging status of VIN is InProgress, battery level: 69
2023-07-26 13:58:50,728 :: INFO :: charging status of VIN is InProgress, battery level: 69
2023-07-26 14:00:50,655 :: INFO :: charging status of VIN is InProgress, battery level: 69
2023-07-26 14:02:50,590 :: INFO :: charging status of VIN is InProgress, battery level: 69
2023-07-26 14:04:50,638 :: INFO :: charging status of VIN is InProgress, battery level: 69
2023-07-26 14:06:50,790 :: INFO :: charging status of VIN is InProgress, battery level: 69
```

**Reply by papageigit** [reply]
Odd, after the car has finished charging, do you experience any economy mode issues? (You step in the car, the screens briefly come on but almost instantly shut off again saying that economy mode is active)

**Reply by maximumstruggle** [reply]
I don't use the car often, our car is for the wife. I don't think she encountered this behaviour as she would have noticed me. The update interval has always been at 2 min since the beginning I use psa_car_controller. It doesn't bother me, I was just wondering if it was possible to manually raising it.

**Reply by stevoh6** [reply]
Maybe, when you calling from home assitant or orher 3rd service psa_controler about current status, you using api without cache -> this force retreive charging status, if you using api with parameter cache you will get correct behaviour :) I had similar problem.

**Reply by bulior** [reply]
@stevoh6 How to set up the API with parametric cache ? And will it update every 15min ?
